flink-connectors/flink-connector-base/src/main/java/org/apache/flink/connector/base/source/reader/fetcher/SingleThreadFetcherManager.java

```diff
@@ -113,6 +113,14 @@ public void addSplits(List<SplitT> splitsToAdd) {
         }
     }
 
+    @Override
+    public void removeSplits(List<SplitT> removedSplits) {
```

Comment:
Would it be more consistent with `addSplits(List<SplitT> splitsToAdd)` to use `splitsToRemove` here?